prestoadmin/topology.py

```diff
@@ -104,9 +106,16 @@ def build_conf_interactive():
     return {'username': prompt_for_username(),
             'port': prompt_for_port(),
             'coordinator': prompt_for_coordinator(),
-            'workers': prompt_for_workers()}
+            'workers': prompt_for_workers(), 
+            'JAVA8_HOME': prompt_for_JAVA8_HOME()}
 
 
+def prompt_for_JAVA8_HOME():
```

Comment:
I don't think it should prompt for JAVA8_HOME actually. it's not super likely the user will have to change this, and we don't want to confuse them. they should still be able to change it via the topology file though